.goreleaser.yml

```diff
@@ -101,7 +101,7 @@ dockers:
     goarch: amd64
     ids:
       - fleetctl
-    dockerfile: tools/docker/fleetctl.Dockerfile
+    dockerfile: tools/fleetctl-docker/Dockerfile
```

Comment:
I can also go the other way around and move `tools/fleetctl-docker/*` into `tools/docker` what's better?

Reply:
I think this is a-ok.

Reply:
I prefer if we put the dockerfile in `tools/docker/fleetctl.Dockerfile`. That way all the dockerfiles are together and we are not mixing conventions ie `tools/docker/fleet.Dockerfile` and `tools/fleetctl-docker/Dockerfile`

Reply:
the flip side is that:

- we already have various `Dockerfile`s all over the place
- this new Dockerfile has a README with details and usage instructions, that's why it's currently in a separate folder

I honestly don't have a strong opinion so happy to change to whatever we agree on.

Reply:
IMO we should keep tools separated by "category/feature" (e.g. `tools/fleetctl-docker` would contain tooling related to fleetctl-docker, `tools/tuf` tooling for TUF repositories, etc.)

Reply:
Is the "category/feature" docker or fleet or fleetctl? If we say that "fleetctl-docker" is a category, then perhaps we should also have "fleet-docker".

Reply:
that makes sense. I pushed a commit